content/using_ec2_spot_instances_with_eks/deploy/tolerations_and_affinity.md

```diff
@@ -4,45 +4,17 @@ date: 2018-08-07T08:30:11-07:00
 weight: 20
 ---
 
-## Adding Tolerations
```

Comment:
Tolerations is still in the release candidate. This would require testing with the latest eksctl version, which I can do with a follow up, once in the release candidate.